Config.lua

```diff
@@ -467,44 +484,21 @@ local spells = {
 	}
 }
 
-for i = 1, MAX_CLASSES do
-	local class = CLASS_SORT_ORDER[i]
-	local name = LOCALIZED_CLASS_NAMES_MALE[class]
-	spells.args[class] = {
-		icon = "Interface\\Icons\\ClassIcon_" .. class,
-		iconCoords = borderlessCoords,
-		name = name,
-		type = "group",
-		get = function(info)
-			return getSpellOption(info)
-		end,
-		set = function(info, val)
-			setSpellOption(info, val)
-		end,
-		args = createOptionsForClass(class)
-	}
-end
 
 local additionalSpellCategories = {
-	["TRINKET"] = "Trinket",
-	["GENERAL"] = "General"
+	["TRINKET"] = INVTYPE_TRINKET,
```

Comment:
I meant to remove L["Trinket"] from the Localization files as the localized version of Trinket is already made by INVTYPE_TRINKET. Just since this is a cleanup PR :D